packages/react-native/scripts/codegen/generate-legacy-interop-components.js

```diff
@@ -18,6 +18,7 @@ const PROJECT_FIELD = 'project';
 const IOS_FIELD = 'ios';
 const LEGACY_COMPONENTS_FIELD = 'unstable_reactLegacyComponentNames';
 const OUTPUT_FILE_NAME = 'RCTLegacyInteropComponents.mm';
+const COMPONENTS_IN_ONE_LINE = 3;
```

Comment:
more than the number of components, it is a matter of number of chars.
I think that the linter will go to a new line if we overflow the 80 or 120 char mark.

Could we refactor the PR with a char-based logic? Because if I have 4 short components:

```
@"RCTA", @"RCTB", @"RCTC", @"RCTD",   
``` 
the internal linter will not put them on multiple lines. :P

Reply:
great point 👍

Reply:
Indeed great point! I've adjusted the implementation 🙌